src/Step/Deploy/InstallUpdate/ConfigUpdate/SetCryptKey.php

```diff
@@ -73,11 +73,6 @@ public function execute(): void
         $key = $this->environment->getCryptKey();
 
         if (empty($key)) {
-            $this->logger->notice(
```

Comment:
Why do you remove the message?

Reply:
equal message presents in `install-update` step and only for upgrade case https://github.com/magento/ece-tools/blob/develop/src/Config/State.php#L95
where it is not only notice, it is exception